src/sentry/services/hybrid_cloud/__init__.py

```diff
@@ -121,21 +124,27 @@ def get_organizations(
 
     @abstractmethod
     def get_organization_by_slug(
-        self, slug: str, only_visible: bool, allow_stale: bool
+        self, user_id: Optional[int], slug: str, only_visible: bool, allow_stale: bool
     ) -> Optional[ApiOrganization]:
         pass
 
-    def _serialize_organization(self, org: Organization) -> ApiOrganization:
+    def _serialize_organization(
+        self, org: Organization, membership: Iterable[OrganizationMember] = tuple()
+    ) -> ApiOrganization:
         return ApiOrganization(
-            slug=org.slug,
-            id=org.id,
+            slug=org.slug, id=org.id, is_member=any(m.organization_id == org.id for m in membership)
```

Comment:
I suppose I'll change this to grabbing the membership object, instead, and serializing it into the `ApiOrganization`.  Also I really want a better name for that model lol.  It's really like, `ThinOrganization`???

Reply:
I think having the membership or its detached instance would be beneficial.